### DISCUSSION TITLE
Code №18 - User was deleted or banned

### DISCUSSION THREAD

**Discussion by vakeanastoiashy** [MrZillaGold/VK2Discord#757] [Q&A] [answered]
Что делать?

**Comment by CloutWarhips** [comment]
**Точно такая же ошибка**

**Reply by egorprnn** [reply]
Попробуйте создать сервисный токен и использовать его

https://github.com/MrZillaGold/VK2Discord/wiki/%D0%A1%D0%B5%D1%80%D0%B2%D0%B8%D1%81%D0%BD%D1%8B%D0%B9-%D1%82%D0%BE%D0%BA%D0%B5%D0%BD

**Comment by egorprnn** [comment, marked as answer]
Попробуйте создать сервисный токен и использовать его

https://github.com/MrZillaGold/VK2Discord/wiki/%D0%A1%D0%B5%D1%80%D0%B2%D0%B8%D1%81%D0%BD%D1%8B%D0%B9-%D1%82%D0%BE%D0%BA%D0%B5%D0%BD
